GatewaySDKFramework/GatewaySDKFramework/zoopHanndler/ZoopHandler.swift

```diff
@@ -51,9 +30,22 @@ class ZoopHandler{
     
     func generateItrURL(ReqObject: ZoopDataObject) -> URL {
         
+        
         var zoopComponents = URLComponents()
         zoopComponents.scheme = "https"
-        zoopComponents.host = "itr.zoop.one"
+        if ReqObject.zoop_env == "QT_P" {
+            zoopComponents.host = "itd.zoop.one"
```

Comment:
Add scheme inside for consistency as anyways the code will be called only in one if block

Reply:
got it doing changes